packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/add/translations/Messages_fr_FR.json

```
@@ -19,5 +19,11 @@
   "kube_nodes_add_flavor_quota_manage": "Gérer mon quota.",
   "kube_nodes_add_error": "Une erreur est survenue lors de l'ajout du nœud : {{ message }}",
   "kube_flavor": "{{ name }} ({{ cpuNumber }} CPU {{ ramCapacity }}Go RAM {{ diskCapacity }}Go Disque)",
-  "kube_flavor_price": "{{price}} HT/h"
+  "kube_flavor_price": "{{price}} HT/h",
+  "kube_nodes_add_title": "{{ nodePoolName }}: Ajouter un nœud",
+  "kube_nodes_add_to_node_pool": "Ajouter des nœuds à un pool de nœuds existant",
+  "kube_nodes_add_same_flavor_message": "Vous pouvez uniquement choisir cette saveur car le pool de nœuds ne peut être composé que de nœuds de la même saveur.",
```

Was this the content provided ? 😄 (saveur means flavor but as a taste like vanilla or strawberry) `
```suggestion
  "kube_nodes_add_same_flavor_message": "Vous pouvez uniquement choisir ce type d'instance car le pool de nœuds ne peut être composé que de nœuds du même type d'instance.",
```